cypress/integration/unit_tests/add_disaster_test.js

```diff
@@ -242,8 +242,7 @@ describe('Unit tests for add_disaster page', () => {
     expect(discrete.children('.box').length).to.equal(2);
     expect(discrete.children().eq(1).css('background-color')).to.equal(red);
 
-    const log = cy.stub(console, 'log')
-                    .withArgs('layer 3: unrecognized color function');
+    const log = cy.stub(console, 'log').withArgs('unrecognized color function');
```

Comment:
do you still want this here? Shouldn't you be asserting on the status?

Reply:
Yes, thanks